### ISSUE TITLE
CSV product image import fails

### ISSUE THREAD

**Issue by therampagerado** [thirtybees/thirtybees#1882]
thirty bees: edge

Today I tried to play with the CSV import functionality, everything seems to work but the images are not imported.

Tried the following:

1. Pulling from another host.
2. Pulling from my host.
3. Pulling from the same host - the URLs are https://www.domain.com/img/path-to-img/img.jpg and they open up in the browser so they are accessible.

If I try to upload multiple image URL values, delimeted by ; the process completes with all green but an error is thrown:
![image](https://github.com/user-attachments/assets/70bb6e4c-29a4-4767-bc2e-774747f558bf)

If I delete most of the URLs and leave only one an error is thrown in the import window: 
![image](https://github.com/user-attachments/assets/1bcf92df-3e4d-4581-b106-246eb92332de)

Even the default file gives an error:
![image](https://github.com/user-attachments/assets/8bbcad61-fab4-4fdc-bb3b-b54b406391d7)

**Comment by getdatakick** [comment]
I can't reproduce it

I tried importing this simple CSV file

```
id_product,image
2,https://img.freepik.com/free-photo/colorful-design-with-spiral-design_188544-9588.jpg;https://letsenhance.io/static/8f5e523ee6b2479e26ecc91b9c25261e/1015f/MainAfter.jpg
```

With `Force all ID numbers` option enabled it updated the existing products - downloaded and associated both images correctly

Maybe filesystem permissions issue? Or missing PHP libraries? Can you upload the images manually in admin product page?

**Comment by therampagerado** [comment]
Thank you for looking into this!

Yes, I'm able to upload the same images manually.
![image](https://github.com/user-attachments/assets/450cbc9b-e544-4afa-829f-230ced0e8a54)

Tried your csv - same issue, the links in the table are made but the images are not downloaded or generated.
![image](https://github.com/user-attachments/assets/f24eb191-6bd6-4939-9958-a7ca89e03f19)

![image](https://github.com/user-attachments/assets/51fe3796-af0a-4424-bd7e-4fe63fa2a2ab)

This definitely worked before on this test install because I deleted the ipods and other test products from the sample file I imported couple of months back.

Now just tried on one of my other installs - same issue, no images, product imports normally.

Where should I check for logs if the problem is on server level?

In the meantime I'll roll back a test install to 1.5 and check with it.
